Makefile

```diff
@@ -149,10 +172,12 @@ doc/doxygen/html/index.html: ${CPPSOURCES} ${PYSOURCES}
 		Doxyfile
 	doxygen
 
+## lib                  : build khmer library
 lib:
```

Comment:
This target is not maintained and should probably be dropped; certainly not advertised.